### PR TITLE
Contribution On Some Essential Tools for Frontend Web Development

### PR DESCRIPTION
**Pull request by Ilsa-Naeem**

### CODE REVIEW

**Review by CodingSpecies — CHANGES_REQUESTED** [review]
@Ilsa-Naeem please revert the changes you made in the .all-contributorssrc file, we will do that ourselves after your pr is merged. Thank you.

**Comment by Ilsa-Naeem** [comment]
@CodingSpecies i have removed the changes made in .all-contributorssrc file

File: .all-contributorsrc [hunk 1]
```diff
@@ -156,16 +156,8 @@
       "contributions": [
         "content"
       ]
-    },
```
Comment (CodingSpecies) [inline .all-contributorsrc]: Please add line 159 back, but remove the comma please @Ilsa-Naeem

**Review by CodingSpecies — CHANGES_REQUESTED** [review]

**Review by TheSlimCoder — APPROVED** [review]
@CodingSpecies I did it by myself @Ilsa-Naeem merging PR.

**Comment by TheSlimCoder** [comment]
@all-contributors add @Ilsa-Naeem for content.

**Pull request merged**